### PR TITLE
APL-1106 fixed array reader.

### PR DESCRIPTION
**Pull request by alzinchenko**

### CODE REVIEW

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/ValueParserImpl.java [hunk 1]
```diff
@@ -12,22 +13,27 @@
 
 @Singleton
 public class ValueParserImpl implements ValueParser {
+    private final String EOT_REGEXP = String.valueOf(CsvAbstractBase.EOT);
```
Comment (silaev-firstbridge) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/ValueParserImpl.java]: Upper case usually indicates a static variable. Have been `static`  forgotten here?

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/ValueParserImpl.java [hunk 2]
```diff
@@ -12,22 +13,27 @@
...
+    private final char quoteChar = CsvAbstractBase.TEXT_FIELD_START;
```
Comment (silaev-firstbridge) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/ValueParserImpl.java]: In turn, this deems to be a static one

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/ValueParserImpl.java (hunk 1 above)
Comment (alzinchenko) [reply apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/ValueParserImpl.java]: sure

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/ValueParserImpl.java (hunk 2 above)
Comment (alzinchenko) [reply apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/ValueParserImpl.java]: sure

**Pull request merged**